05-how-to-contribute-to-open-source.md

```diff
@@ -170,6 +170,7 @@ You'll need to have these tools donwloaded and installed on your local machine:
     ```
 
 11. Go to your forked repository on GitHub and create a pull request with the title `feat: Add <@github-username> as a contributor`.
+   !> Your pull request will be marked as invalid and may be closed if the form is incomplete.
```

Comment:
We want to apply the suggestion on the issue. So, can you please:

1. Update the wording in point 11.
2. Add space for the alert.

```suggestion
11. Go to your forked repository on GitHub. Create a pull request with the title `feat: Add <@github-username> as a contributor` and fill in all areas in the pull request form.

   !> Your pull request will be marked as invalid and may be closed if the form is incomplete.
```